HW_1.2.py

```diff
@@ -0,0 +1,8 @@
+a_rubl = int(input())
+b_coin = int(input())

```

Comment:
it's not a part of homework1 but homework 2. Sanity check failed - redundant code in PR